src/App.js

```diff
@@ -42,6 +42,11 @@ export default function App() {
   const [allTransactions, setAllTransactions] = useState([]);
   const [lastUpdate, setLastUpdate] = useState('');
   const [prependFilenameWithAcctNames, setPrependFilenameWithAcctNames] = useState(true);
+  const [divisorPower, setDivisorPower] = useState(9); // ONEDAY: read from localStorage
+  const divisorPowerOptions = [[0, 9].map((x) => ({ value: x, label: x ? `1e${x}` : 1 }))].flat();
+  const [decimalPlaces, setDecimalPlaces] = useState(6); // ONEDAY: read from localStorage
+  const decimalPlacesOptions = [[1, 2, 3, 4, 5, 6, 7, 8].map((x) => ({ value: x, label: x }))].flat();
```

Comment:
Out of curiosity why to do this instead of:
[1, 2, 3, 4, 5, 6, 7, 8].map((x) => ({ value: x, label: x })) directly?

Same question for line 46.

Reply:
Good catch. This was accidental (added by Copilot, and I forgot to clean it up).

I just cleaned it up: https://github.com/NEARFoundation/tx-tracking-app/pull/12/commits/af9b7bc51e8388312f61878454b91f490be443f2